### DISCUSSION TITLE
Disabling background hotkeys when we open a modal.

### DISCUSSION THREAD

**Discussion by bytesagar** [JohannesKlauss/react-hotkeys-hook#1180] [Q&A]
Can we disable the background hotkeys used when we are inside any modals?.
I have hotkeys in sidebar that toggle the sidebar tabs menu. When I open any modal in the current page. I want to disable those keys in the sidebar and prevent interaction in background. Is there anyway?

**Comment by JohannesKlauss** [comment]
This hook is modular, so you have to implement that in user land. Use a global state that holds a flag of the modal state (active or not active). Then use that flag to determine if a hotkey should be enabled by setting that flag on the `enabled` option of the hotkey.
